### DISCUSSION TITLE
Hitbox ✅ UI ❌

### DISCUSSION THREAD

**Discussion by Skibidi1927** [ppy/osu#26355] [Ideas]
![IMG_6143](https://github.com/ppy/osu/assets/125677824/3a8f7931-1f5a-4eb8-8b5c-14eb21f8ce6d)
Hitbox are now playable but ui interface make it unplayable

**Comment by nagi-desuuu** [comment]
Seems like a duplicate of https://github.com/ppy/osu/issues/25852
